lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -3,9 +3,9 @@
 _template:
   context: ["interface <name>"]
   ios_xr:
-    get_command: "show running interface"
+    get_command: "show running interface <name>"
```

Comment:
I would not make any updates to XR.  They are not needed at this point and we have discussed just removing the XR support.

Reply:
ack